### PR TITLE
Support label when creating and unlocking encrypted devices

### PR DESCRIPTION
**Pull request by vojtechtrefny**
Fixes: #1002

### CODE REVIEW

File: src/udiskslinuxblock.c [hunk 1]
```diff
@@ -3445,6 +3452,7 @@ udisks_linux_block_handle_format (UDisksBlock             *block,
   g_variant_lookup (options, "encrypt.iterations", "u", &encrypt_iterations);
   g_variant_lookup (options, "encrypt.time", "u", &encrypt_time);
   g_variant_lookup (options, "encrypt.threads", "u", &encrypt_threads);
+  g_variant_lookup (options, "encrypt.label", "&s", &encrypt_label);
```
Comment (tbzatek) [inline src/udiskslinuxblock.c:3455]: Would you document this new argument in the D-Bus docs please?

Comment (vojtechtrefny) [reply src/udiskslinuxblock.c:3455]: I knew I forgot something. Fixed.

Comment (vojtechtrefny) [reply src/udiskslinuxblock.c:3455]: Fixed again :man_facepalming:

**Review by tbzatek — APPROVED** [review]
Good to go now ;-)

**Pull request merged**